test/specs/modules/Modal/Modal-test.js

```diff
@@ -45,6 +45,8 @@ describe('Modal', () => {
   })
 
   common.hasSubComponents(Modal, [ModalHeader, ModalContent, ModalActions, ModalDescription])
+  common.hasValidTypings(Modal)
```

Comment:
Looks like this might already be covered by the `common.isConformant()` test, true?